ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CassAstyanaxPlugin.java

```diff
@@ -181,6 +184,34 @@ public void shutdown() throws Exception {
 
     }
 
+    /**
+     * Perform a bulk read operation
+     * @return a list of response codes
+     * @throws Exception
+     */
+    public List<String> readBulk(final List<String> keys) throws Exception {
```

Comment:
The implementation might be different in C* instead of merging single read operation. There is multiget thrift operation which is more suitable in this case, hence, I would leave the implementation to null and fix later.

Reply:
+1. The only plugin that leverage the bulk api would be the janusgraph. i will leave the rest of the plugins as null impl and throw an exception if triggers

Reply:
#resolved